ctx.go

```diff
@@ -901,6 +902,34 @@ func (c *Ctx) QueryParser(out interface{}) error {
 	return decoder.Decode(out, data)
 }
 
+// ReqHeaderParser binds the request header strings to a struct.
+func (c *Ctx) ReqHeaderParser(out interface{}) error {
+	// Get decoder from pool
+	decoder := decoderPool.Get().(*schema.Decoder)
+	defer decoderPool.Put(decoder)
+
+	// Set correct alias tag
+	decoder.SetAliasTag(reqHeaderTag)
+
+	data := make(map[string][]string)
```

Comment:
is it possible to pass in the part from outside so that the outer processing that is identical for any parsing like queryParser, bodyParser etc. can be reused?
```go
func (c *Ctx) parserToStruct(out interface{}, data map[string][]interface{}) error {
```
something like that ?